grpc/pb/space.proto

```diff
@@ -95,4 +106,66 @@ message CreateFolderRequest {
 }
 // not sure we need to return anything other than an error if we failed
 message CreateFolderResponse {
+}
+
+message GetIdentityByUsernameRequest {
+  string username = 1;
+}
+
+message Identity {
+  string address = 1;
```

Comment:
It's another type of address based on the public key (just like a crypto address in blockchain). This is just in case you want to send a message to a given public key and you don't have the username.

The address book models look like this:

Identity Model: address (pub key representation), username
Proof model: proof type (e.g. email, twitter handle), username

Identity has many proofs.